types/node/crypto.d.ts

```diff
@@ -120,7 +120,7 @@ declare module "crypto" {
     function createHash(algorithm: string, options?: HashOptions): Hash;
     function createHmac(algorithm: string, key: BinaryLike | KeyObject, options?: stream.TransformOptions): Hmac;
 
-    type Utf8AsciiLatin1Encoding = "utf8" | "ascii" | "latin1";
+    type Utf8AsciiLatin1Encoding = "utf8" | "utf16le" | "ascii" | "latin1";
```

Comment:
I think we need to update the type name as it doesn't reflect what it contains anymore.

Reply:
Agreed. Reading the documentation it would seem that these are all interchangeable. A quick test also seems to prove it right:

![Screen Shot 2020-09-15 at 2 50 01 PM](https://user-images.githubusercontent.com/347002/93268859-cdcf6c00-f762-11ea-84e7-c0a596e689ce.png)

Where these types are trying to be restrictive and only allow certain encodings for certain methods, Node seems to allow them all.